src/components/CreatePost/CreatePost.jsx

```diff
@@ -0,0 +1,191 @@
+                    }
+                )
+
+                if(updatedDataResponse.data.status==="ok")
+                {
+                    dispatch(updateHomeFeed(updatedDataResponse.data.homefeed.allHomeFeedPosts))
+                    setNewPostTextContent("")
+                    setPostCharCount(280)
+                }
+            }
+        }
+        else
+        {
+            showToast("warning","Kindly login to post!")
+        } 
+    }
+
+    return (
+        <div className="create-new-post-container">
+
+          <div className="avatar avatar-small">
+              {
+                userLoggedIn ?
+                (
+                    <img className="avatar-img" src={loggedInUserProfilePic} alt="avatar"/>
+                ) 
+                : (
+                    <img className="avatar-img" src="https://enztron-dev-branch.netlify.app/Icons-and-Images/Avatars/blue-illustration-avatar.svg" alt="avatar"/>
+                )
+              }
+              <span className="status-badge-2x status-online"></span>
+          </div>
+
+          <div className="create-new-post">
+            <textarea className='whats-happening'  
+            cols="50" 
+            placeholder={`What's on your mind${inputTextareaPlaceholder}?`}
+            value={newPostTextContent}
+            onChange={event => {
+                setPostCharCount(280-event.target.value.length)
+                setNewPostTextContent(event.target.value)
+            }}>
+              
+            </textarea>
+
+            <hr></hr>
+            <div className="create-post-options-container">
+              
+              <div className="create-post-options">
+                  <div className="create-post-icons-container">
+                    <BiImageAdd className="create-new-post-icons"/>
+                  </div>
+                  <div className="create-post-icons-container">
+                    <AiOutlineFileGif className="create-new-post-icons"/>
+                  </div>
+                  <div className="create-post-icons-container">
+                    <BiPoll className="create-new-post-icons"/>
+                  </div>
+                  <div className="create-post-icons-container">
+                    <BsEmojiSmile className="create-new-post-icons"/>
+                  </div>
+                  <div className="create-post-icons-container">
+                    <IoCalendarOutline className="create-new-post-icons"/>
+                  </div>
+                  <div className="create-post-icons-container">
+                    <BiMap className="create-new-post-icons"/>
+                  </div>
+              </div>
+
+              <div className="post-options-right-container">
+                {
+                  postCharCount<0 ?
+                    (<p style={{borderColor:"var(--notification-or-error)",color:"var(--notification-or-error)"}} className="new-post-ch-count">{postCharCount}</p>)
```

Comment:
Suggestion: it would better if you could avoid inline styling

Reply:
Done